model/src/test/kotlin/utils/FindingsMatcherTest.kt

```diff
@@ -86,6 +86,59 @@ class FindingsMatcherTest : WordSpec() {
             }
         }
 
+        "Given a license finding in a license file and a file with copyright and license findings, match" should {
+            "not associate that copyright with the root license" {
+                setupLicenseFinding(license = "some id", path = "a/LICENSE")
+                setupLicenseFinding(license = "some other id", path = "some/other/file")
+                setupCopyrightFinding(statement = "some stmt", path = "some/other/file")
+
+                val result = matcher.match(licenseFindings, copyrightFindings)
+
+                result.getFindings("some id").copyrights should beEmpty()
+            }
+        }
+
+        "Given license findings in two license files a copyright finding in a file without license, match" should {
+            "associate that copyright with excatly one root license" {
```

Comment:
Looking at the assertions below, shouldn't that be "associate that copyright with both root licenses"?

Reply:
Actually this test failed and it failed because it found a bug.
I've adjusted the test to test the implemented behavior.
I will fix the bug later on.